src/app/utilities/registerSubscriptionData.ts

```diff
@@ -0,0 +1,28 @@
+import axios from 'axios';
+import UserSubscriptionInterface from '../interfaces/UserSubscriptionInterface';
+import CRMCreateResponseInterface from '../interfaces/CRMCreateResponseInterface';
+
+export const registerSubscriptionData = async (
+  userSubscriptionData: UserSubscriptionInterface
+): Promise<CRMCreateResponseInterface> => {
+  const data = JSON.stringify(userSubscriptionData);
+
+  const config = {
+    method: 'post',
+    url: 'https://api.hubapi.com/crm/v3/objects/contacts',
```

Comment:
Can you upgrade the existing registerContact function to accept subscription information rather than creating a new one?

Reply:
Are we going to have similar registration procedures for contacts & subscriptions in the longer term? If not, having two distinct functions shouldn't hurt.